packages/web/components/Guild/GuildForm.tsx

```diff
@@ -197,7 +204,7 @@ export const GuildForm: React.FC<Props> = ({
             isInvalid={!!errors.name}
             background="dark"
           />
-          <span>
+          <span className="description">
```

Comment:
If you want to repeat the style, create a local component like:

```typescript
const Description: React.FC = ({ children }) => (
  <Text as="span" ml={4} mt={1} fontSize="sm">
    {children}
  </Text>
);
```
```suggestion
          <Description>
```